### DISCUSSION TITLE
Convert 24 bit FLAC to 16 bit

### DISCUSSION THREAD

**Discussion by strange-antiquark** [enzo1982/freac#349] [Help & Support] [answered]
I cannot find any way to convert a file @SUBJ. If there isn't a way, this would be a rather nice feature to have -- far down on the list of priorities, I realize, but something perhaps to consider.

**Comment by enzo1982** [comment, marked as answer]
To convert a 24 bit FLAC to 16 bit, enable the _Sample Format Converter_ in the _Processing_ settings (reachable via the small blue button at the bottom right of the main window).

In the _Sample Format Converter_ configuration, select _Integer_, _16 bit_ as the target format.

With the regular output file settings, you will get new 16 bit FLAC files in addition to your 24 bit source files. If you would like to replace your source files with 16 bit versions instead, change the configuration like this:

- On the _Encoders_ page of the configuration dialog, enable the _Use input file folder if possible_ and _Allow overwriting input file_ options.
- On the same page, set the _Filename pattern_ option to `<filename>` and make sure the _Append sequential numbers..._ option is unchecked.

**Reply by strange-antiquark** [reply]
Many thanks, this does the trick.
I also found it necessary to enable sample rate conversion -- 192K/s is simply ridiculous.

**Reply by markruys** [reply]
The order seems to be important: first downsample to 44.1 kHz, then downsample to 16 bit resolution.

**Comment by zordaz** [comment]
Does Fre:ac apply dithering during conversion from 24 to 16 bits (it should of course)? If so, what type of dithering?

@strange-antiquark : not only is 192K ridiculous, the playback fidelity in fact is even slightly worse compared to 16/44.1.... But the marketing departments makes people think that bigger & higher is always better...

**Reply by strange-antiquark** [reply]
What's even worse is that there are people who should know that bigger and higher isn't better, but have convinced themselves that their ears are special, and they can hear frequencies that even a bat cannot :D

I can't answer your question, of course; all I know is that I am very very satisfied with Fre:ac's performance. I'm learning new tricks every time I use it, and right now, it's getting a great deal of use.

**Reply by enzo1982** [reply]
> Does Fre:ac apply dithering during conversion from 24 to 16 bits (it should of course)? If so, what type of dithering?

It didn't so far, but I implemented it today and it's now available in the [continuous builds](https://github.com/enzo1982/freac/releases/tag/continuous).

Dithering is enabled by default and defaults to TDPF dither. RPDF is available as an option.
